pallets/liquidation-pools/src/lib.rs

```diff
@@ -325,6 +339,15 @@ pub mod module {
 			target_amount: Balance,
 		) -> DispatchResultWithPostInfo {
 			let _ = ensure_none(origin)?;
+			ensure!(
+				T::LiquidityPoolsManager::pool_exists(&supply_pool_id),
+				Error::<T>::PoolNotFound
+			);
+			ensure!(
+				T::LiquidityPoolsManager::pool_exists(&target_pool_id),
+				Error::<T>::PoolNotFound
+			);
+
```

Comment:
```suggestion
			ensure!(
				T::LiquidityPoolsManager::pool_exists(&supply_pool_id)
					&& T::LiquidityPoolsManager::pool_exists(&target_pool_id),
				Error::<T>::PoolNotFound
			);
```
This suggestion is optional :)

Reply:
Done